language/tools/move-package/src/source_package/parsed_manifest.rs

```diff
@@ -55,15 +56,31 @@ pub struct InternalDependency {
     pub subst: Option<Substitution>,
     pub version: Option<Version>,
     pub digest: Option<PackageDigest>,
+    pub dep_override: Option<DepOverride>,
 }
 
-#[derive(Debug, Clone, Eq, PartialEq)]
+#[derive(Debug, Clone, Eq)]
 pub enum DependencyKind {
     Local(PathBuf),
     Git(GitInfo),
     Custom(CustomDepInfo),
 }
 
+/// Custom implementation to normalize local paths
+impl PartialEq for DependencyKind {
+    fn eq(&self, other: &Self) -> bool {
+        match (self, other) {
+            (&DependencyKind::Local(ref p), &DependencyKind::Local(ref op)) => {
+                normalize_path(p, true).unwrap() == normalize_path(op, true).unwrap()
```

Comment:
Is it possible to ensure local paths are stored normalized?  No worries if not, but I remember some other occasions where I ran into this, and that's how I solved it (by calling `normalize_path`).

Reply:
> Is it possible to ensure local paths are stored normalized?

That was my first thought, but this I believe would require changing the code parsing the manifest file and I wasn't sure we wanted to go there. The comparison change seemed less invasive, but I am happy to go the other route if it looks better.

Reply:
Let's settle on normalizing local paths.